### PR TITLE
feature: improve UX when auto paste failed

### LINKED ISSUE #1610: [Demand] A better UI on auto paste failed
Now we're using prompt which is not ergonomic for users.

### PR DESCRIPTION
**Pull request by Jack-Works**
close #1610 

Before:

New:

### CODE REVIEW

File: src/components/InjectedComponents/AutoPasteFailedDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import { DialogDismissIconUI } from './DialogDismissIcon'
+import { Image } from '../shared/Image'
+import { DraggableDiv } from '../shared/DraggableDiv'
+
+export interface AutoPasteFailedDialogProps extends withClasses<KeysInferFromUseStyles<typeof useStyles>> {
+    onClose: () => void
+    data: MaskbookMessages['autoPasteFailed']
+}
+const useStyles = makeStyles({
+    title: { marginLeft: 8 },
```
Comment (guanbinrui) [inline src/components/InjectedComponents/AutoPasteFailedDialog.tsx:25]: `theme.spacing`

**Comment by guanbinrui** [comment]
It looks like the image payload failed to decode if I copied it from the "Paste Manually" dialog.

**Comment by Jack-Works** [comment]
Strange

**Comment by guanbinrui** [comment]
LGTM

**Review by septs — APPROVED** [review]

**Review by guanbinrui — APPROVED** [review]

**Comment by Jack-Works** [comment]
We may unlock image payload for the mobile cause there is a fallback for this (download & re-upload)

**Pull request merged**